server/sonar-web/src/main/js/apps/account/organizations/CreateOrganizationForm.js

```diff
@@ -83,19 +86,19 @@ class CreateOrganizationForm extends React.PureComponent {
     this.closeForm();
   };
 
-  handleAvatarInputChange = (e: Object) => {
+  handleAvatarInputChange = (e /*: Object */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)